tests/jballerina-unit-test/src/test/resources/test-src/expressions/access/field_access.bal

```diff
@@ -353,10 +353,50 @@ function testFieldAccessOnMapConstruct() returns boolean {
     return "Sanjiva" == name;
 }
 
+class EmployeeR {
+    function func(int i) returns int => i;
+}
+
+class ManagerR {
+    function func(int i) returns int => i + 1;
+}
+
+class CompanyR {
+    function func(int i) returns string => (i + 2).toString();
+}
+
+function testAccessingMethodOnUnionObjectType() {
```

Comment:
Where is this invoked from?

Reply:
Updated 79d7ed91adf3807fa51e7ae00800fa937d4ea30a